doc/cygnus-common/backends_catalogue/orion_backend.md

```diff
@@ -2,32 +2,36 @@
 ## `OrionBackend` interface
 This class enumerates the methods any [Orion](https://github.com/telefonicaid/fiware-orion) backend implementation must expose. In this case, the following ones:
 
-    JsonResponse subscribeContextV1(String cygnusSubscription, String token) throws Exception;
-    
-> Subscribes to Orion given a Json Cygnus subscription (Orion subscription + Orion endpoint) and a token for authentication purposes (NGSIv1).
-    
     JsonResponse subscribeContextV2(String cygnusSubscription, String token) throws Exception;
```

Comment:
@IvanHdzC I think we could do the following change in name methods in the code:

subscrbieContextV2 -> subscrbieContext

as they are the  "oficial" version of the standard so they shouldn't need any suffix/prefix.

In addition, if I remember correctly Java has a documentation mark for deprecated stuff  (`@deprecated` ?). The old V1 methods could be marked with it.

What do you think?

Reply:
Discussed F2F @fgalan and @IvanHdzC 

With regards to the "subscribeContextV2 -> subscrbieContext" changes: not going to be done.

With regards to `@deprecated`, is going to be done. @IvanHdzC takes this task.

Reply:
NTC in the context of this PR. The pending pieces of work has been moved to an idependent issue: https://github.com/telefonicaid/fiware-cygnus/issues/1775